src/widget/container/mod.rs

```diff
@@ -114,7 +118,36 @@ where
         origin: Point,
         size: Size,
     ) -> Result<(), Display::Error> {
-        let mut y: i32 = 0;
+        let num_children = self.options.children.len() as u32;
+
+        if num_children == 0 {
+            return Ok(());
+        }
+
+        let total_children_height = self.content_size().height.unwrap_or(0);
+        let unused_height = size.height - total_children_height;
+
+        let (mut current_y, space) = match self.options.justification {
```

Comment:
- current_y: y position to draw the next child at
- space: space between the children